src/components/GlobalPreferences/Network/Network.js

```diff
@@ -89,6 +104,50 @@ function Network() {
             `}
           />
         </Label>
+        <Label theme={theme}>
+          Subgraph HTTP Endpoint
+          <TextInput
+            value={subgraphHttpEndpoint}
+            wide
+            onChange={handleSubgraphHttpEndpointChange}
+            css={`
+              ${textStyle('body2')};
+              color: ${theme.contentSecondary};
+            `}
+          />
+          {httpSubgraphError && (
+            <span
+              css={`
+                ${textStyle('body4')};
+                color: ${theme.negative};
+              `}
+            >
+              {httpSubgraphError}
+            </span>
+          )}
+        </Label>
+        <Label theme={theme}>
```

Comment:
We're duplicating these same sets of components multiple times... so might be a good opportunity to create a new component :).